<launch>


  <!-- Core agent vision nodes --> 
  <node pkg="rapp_ros_naoqi_wrappings" name="acore_camera_server" type="acore_camera_server.py" output="screen"/>
  <node pkg="rapp_ros_naoqi_wrappings" name="acore_adyn_qrcode_server" type="adyn_qrcode_server_navigation.py" output="screen"/>   <!-- QR code detection server-->

  <!-- Core agent navigation nodes -->
  <node pkg="rapp_ros_naoqi_wrappings" name="acore_move_server" type="acore_move_server.py" output="screen"/>
   <node pkg="rapp_ros_naoqi_wrappings" name="acore_execution_server" type="acore_execution_server.py" output="screen"/>
   <node pkg="rapp_ros_naoqi_wrappings" name="acore_obstacle_server" type="acore_obstacle_server.py" output="screen"/>
   <node pkg="rapp_ros_naoqi_wrappings" name="acore_estimator_server" type="acore_estimator_server.py" output="screen"/>
   <node pkg="rapp_ros_naoqi_wrappings" name="acore_state_server" type="acore_state_server.py" output="screen"/>
  <node pkg="rapp_ros_naoqi_wrappings" name="acore_transform_server" type="acore_transform_server.py" output="screen"/>


  <!-- Kalman filter node  -->
  <include  file="/home/nao/ws_rapp_nao/install_isolated/share/rapp_ros_naoqi_wrappings/launch/navigation/nao_ekf.launch" />


</launch>
